crates/jstzd/tests/octez_node_test.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg(not(target_os = "macos"))]
+// This test is disabled for macos because octez binary is not available
+
+use jstzd::task::{octez_node, Task};
+use rand::Rng;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+#[ignore = "ignore for now until octez binaries in CI are set up properly"]
+#[tokio::test(flavor = "multi_thread")]
+async fn octez_node_test() {
```

Comment:
things like creating tmp dir with prefix and generating random ports will likely be reused. I think it would make sense to factor into a common module.

Reply:
Indeed, but let's do this later on when we are reusing these pieces of code.

Reply:
Ok, tracked here: https://linear.app/tezos/issue/JSTZ-151/factor-out-common-utility-functions